### PR TITLE
Add print button 

### PR DESCRIPTION
**Pull request by nikolaiwarner**
- and hide map on mobile in all cases
- and clean up print css to look good again after the recent flexboxing of things

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -64,9 +64,12 @@
           <div id="facets" class="tab-pane control control-survey active"></div>
           <div id="listTab" class="tab-pane control control-sidebar">
             <div class="list-filters">
-              <select id="select_county">
+              <select id="select_county" class="form-control">
                 <option value="" selected>All Counties</option>
               </select>
+              <div class="btn btn-default btn-print" title="Print" onClick="window.print();">
+                🖶 Print
```
Comment (chaserx) [inline index.html]: is there a symbol there (before Print) that won't show up here?

**Comment by chaserx** [comment]
+1 upon addressing previous comment

File: index.html (hunk 1 above)
Comment (nikolaiwarner) [reply index.html]: Hmmm. I can see it. It's the utf8 printer icon .... maybe this is safer? &#128438;

**Comment by eeeschwartz** [comment]
oh heck yeah. looks real nice +1

**Pull request merged**